### DISCUSSION TITLE
Docker install and HTTPS?

### DISCUSSION THREAD

**Discussion by jwalsh2me** [wavelog/wavelog#958] [Q&A] [answered]
I was wondering if folks are running the Docker installation in a hosting provider and if you are how are you handling SSL?  I was thinking of using a NGINX container to do the redirect and renew the letsencrypt cert but was curious as to what others might be doing.

John W9BLN

**Comment by berrySkies** [comment]
Hi John
I was asking almost the same question below in this [link](https://github.com/wavelog/wavelog/discussions/956). This SSL stuff and docker is a bit beyond me. If you decide on an approach would be great if you could post here.
Thanks Alan VK2ATW

**Reply by jwalsh2me** [reply]
Right now I am leaning towards using Caddy [link](https://github.com/lucaslorentz/caddy-docker-proxy) to do the cert renewal and HTTPS proxy. I will let you know how it goes, last night I got it working on my laptop now to move it to Linode and see if I can get it all going there.

John W9BLN

**Comment by violarulan** [comment, marked as answer]
The usual way to do this is to put wavelog behind a reverse proxy (nginx or caddy or something). We let the Wavelog container to listen to the local network and the reverse proxy to handle the HTTPS stuff.

**Reply by berrySkies** [reply]
Thanks for advice....I'll try Caddy as mentioned by John also.

**Comment by jwalsh2me** [comment]
Thanks, I ended up using Caddy.  It was so simple and clean.
-- 
John Walsh
***@***.***
